ignition/service/requestqueue.py

```diff
@@ -207,6 +209,7 @@ def handle_request(self, request):
             request_as_dict['system_properties'] = PropValueMap(request_as_dict['system_properties'])
             request_as_dict['request_properties'] = PropValueMap(request_as_dict['request_properties'])
             request_as_dict['associated_topology'] = AssociatedTopology.from_dict(request_as_dict['associated_topology'])
+            request_as_dict['version'] = REQUEST_MESSAGE_VERSION
```

Comment:
Believe this should be `request_as_dict['version'] = request_as_dict['version']`

Reply:
Which means we can just remove this line?

Reply:
Addressed in commit 5.